rfcs/0000-rfc-template.md

```diff
@@ -23,6 +24,13 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 
 ## Fields
 
+The `rule` fields being proposed are as follows:
+
+Field | Type | Description /Usage
+-- | -- | -- | -- | -- 
+rule.tags | array | Used to track the set of tags applied to a rule | Customers can use it to indicate: author, benchmark partial name, rule number, rule category etc. It will be useful when we extend the capability to add more rules
```

Comment:
ECS already provides a top-level `tags` field: https://www.elastic.co/guide/en/ecs/current/ecs-base.html#field-tags.

Could the existing `tags` field be used for the same purpose without creating another `tags` field?

Reply:
This could be the right approach to reuse top-level tags to include rules tag.